cleanup_k8s.py

```diff
@@ -0,0 +1,13 @@
+import logging
+from ocw.lib.gke import GKE
+from webui.settings import PCWConfig
+
```

Comment:
not sure what different it makes after all but nevertheless I would wrap this into standard `main` method 

e.g. 
```
def main():
    print("Hello, World!")
    if __name__== "__main__" :
main()
```

Reply:
done